README.md

```diff
@@ -166,6 +167,62 @@ mysql::db { 'mydb':
 }
 ```
 
+### Install Percona server on Centos
+
+This example shows how to do a minimal installation of a Percona server on a Centos (has been tested on Puppet 4.4 / Centos 7 / Percona Server 5.7) including the Percona server, client and bindings.
+
+The configuration change here is really minimal.
+
+**Note:** The installation of the yum repository is not part of this package and is here only to show a full example of how you can install this.
+
+```puppet
+yumrepo { 'percona':
+  descr    => 'CentOS $releasever - Percona',
+  baseurl  => 'http://repo.percona.com/centos/$releasever/os/$basearch/',
+  gpgkey   => 'http://www.percona.com/downloads/percona-release/RPM-GPG-KEY-percona',
+  enabled  => 1,
+  gpgcheck => 1,
+}
+
+class {'mysql::server':
+  package_name     => 'Percona-Server-server-57',
+  service_name     => 'mysqld',
+  config_file      => '/etc/my.cnf',
+  includedir       => '/etc/my.cnf.d',
+  override_options => {
+    mysqld => {
+      log-error => '/var/log/mysqld.log',
+      pid-file  => '/var/run/mysqld/mysqld.pid',
+    },
+    mysqld_safe => {
+      log-error => '/var/log/mysqld.log',
+    },
+  }
+}
+
+# Note: this is not needed  in our case as installing Percona-Server-server-57
+# also installs Percona-Server-client-57. It is only here to show how to
+# install the Percona MySQL client
+class {'mysql::client':
+  package_name => 'Percona-Server-client-57',
+}
+
+class { 'mysql::bindings':
+  client_dev_package_name => 'Percona-Server-devel-57',
+  daemon_dev_package_name => 'Percona-Server-shared-57',
```

Comment:
Add these since they default to `false`
```
  client_dev => true,
  daemon_dev => true,
```

in addition to those it may make sense to install the Perl and Python bindings.

```
    perl_enable               => true,
    python_enable             => true,
    perl_package_name         => 'perl-DBD-MySQL',
    python_package_name       => 'MySQL-python',
```